view_db.py

```
@@ -0,0 +1,32 @@
+    cursor.execute("SELECT name FROM sqlite_master WHERE type='table';")
+    tables = cursor.fetchall()
+    
+    print("\n=== Database Contents ===\n")
+    
+    for table in tables:
+        table_name = table[0]
+        print(f"\nTable: {table_name}")
+        print("-" * 80)
+        
+        # Read table into pandas DataFrame
+        df = pd.read_sql_query(f"SELECT * FROM {table_name}", conn)
```

Ruff (0.13.3)

Possible SQL injection vector through string-based query construction

(S608)